ledger/participant-integration-api/src/test/suite/scala/platform/indexer/parallel/ParallelIndexerSubscriptionSpec.scala

```diff
@@ -3,24 +3,50 @@
 
 package com.daml.platform.indexer.parallel
 
-import java.time.Instant
-
 import com.codahale.metrics.MetricRegistry
 import com.daml.ledger.offset.Offset
 import com.daml.ledger.participant.state.{v2 => state}
-import com.daml.lf.data.Ref
+import com.daml.lf.crypto.Hash
 import com.daml.lf.data.Time.Timestamp
+import com.daml.lf.data.{ImmArray, Ref, Time}
+import com.daml.lf.transaction.TransactionNodeStatistics.EmptyActions
+import com.daml.lf.transaction.{
+  CommittedTransaction,
+  TransactionNodeStatistics,
+  TransactionVersion,
+  VersionedTransaction,
+}
 import com.daml.logging.LoggingContext
 import com.daml.metrics.Metrics
 import com.daml.platform.indexer.parallel.ParallelIndexerSubscription.Batch
 import com.daml.platform.store.backend.{DbDto, ParameterStorageBackend}
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
+import java.time.Instant
+
 class ParallelIndexerSubscriptionSpec extends AnyFlatSpec with Matchers {
 
   private implicit val lc: LoggingContext = LoggingContext.ForTesting
 
+  private val DbDtoEq: org.scalactic.Equality[DbDto] = {
```

Comment:
this is a copy paste of what we have at UpdateToDbDtoSpec. would you consider to add this to some test-lib-support object and use it from there both places?

I like the idea a lot to have unit tests for the test-comparator

Reply:
I've refactored DbDtoEq into it's own class and removed it from other places..